docs/source/programs_and_gates.rst

```diff
@@ -685,49 +702,3 @@ All of the above methods will produce the same output:
 
    X 0
    Y 1
-
-The ``pyquil.parser`` submodule provides a front-end to other similar parser
-functionality.
-
-
-Fixing a Mistaken Instruction
```

Comment:
The `pop` instruction will be removed, and I'm not sure this is totally helpful outside of a REPL environment. Happy to replace this with a working example though, if we think it should be carried forward.